targets/TARGET_NORDIC/TARGET_NRF5x/gpio_api.c

```diff
@@ -129,8 +129,18 @@ static void gpio_apply_config(uint8_t pin)
             cfg.hi_accuracy = false;
             cfg.is_watcher = false;
             cfg.sense = NRF_GPIOTE_POLARITY_TOGGLE;
+            switch(m_gpio_cfg[pin].pull) {
```

Comment:
`m_gpio_cfg[pin].pull` should be initialized to `NRF_GPIO_PIN_NOPULL` in function `gpio_irq_init` below.